### PR TITLE
Define default role for links in sphinx

### PR DESCRIPTION
**Pull request by apojomovsky**
This goes hand-by-hand with [ros2_documentation#5](https://github.com/ros2/ros2_documentation/pull/5), as this enable the internal links to work from the `.rst` files.

### CODE REVIEW

File: _sphinx/conf.py [hunk 1]
```diff
@@ -45,6 +45,9 @@
 # The full version, including alpha/beta/rc tags.
 release = u''
 
+# -- Define the default role to use for links ----------------------------
```
Comment (hidmic) [inline _sphinx/conf.py]: @apojomovsky nit: can we take those dashes out so that it looks like the rest?

**Review by hidmic — APPROVED** [review]
LGTM!

File: _sphinx/conf.py (hunk 1 above)
Comment (apojomovsky) [reply _sphinx/conf.py]: sure, done!

**Comment by apojomovsky** [comment]
Comments addressed! I'm moving forward and merging this one now. Thanks!

**Pull request merged**